code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -974,10 +974,53 @@
 			else
 				message = "makes a light spitting noise, a poor attempt at a whistle."
 
+		if ("roll")
+			if(!param)
+				to_chat(src, "To use this emote specify what die you want to roll in the first line. Prefix it with a \"D\". You may add a bonus to your roll, simply add a \"+\" and your number after your chosen die. If you want to incorporate a difficulty check, add a \"-DC\". You may use a bonus, a difficulty check or both.<br>Here is an example: roll-D20+5-DC15. This will roll a twenty-sided die, add five and compare it against a difficulty check of 15.")
```

Comment:
this usage text should be a variable you declare before the if(!param) and used here and at the if(!message), to avoid having to adjust it multiple times (its also better style)